tactics/cbn.ml

```diff
@@ -449,7 +449,7 @@ let magically_constant_of_fixbody env sigma (reference, params) bd = function
             let addus l r (qs,us) = qs, Univ.Level.Map.add l r us in
             let subst = Set.fold (fun cst acc ->
                 match cst with
-                | QEq (a,b) | QLeq (a,b) ->
+                | QEq (a, b) | QLeq (a, b) | QElimTo (a, b) ->
```

Comment:
I don't understand how this can make sense. We're not supposed to add elimination constraints when equating terms, are we? If so this should be an assertion failure (with a comment).